__tests__/api.test.js

```diff
@@ -59,4 +59,45 @@ describe("Returns with the correct JSON endpoint", () => {
         expect(input).toMatchObject(expectedOutput);
       });
   });
+  test("/api/reviews/:review_id", () => {
+    const review_id = 3;
+    return request(app)
+      .get(`/api/reviews/${review_id}`)
+      .expect(200)
+      .then((response) => {
+        response.body.review.forEach((review) => {
+          expect(typeof review.title).toBe("string");
+          expect(typeof review.created_at).toBe("string");
+          expect(typeof review.votes).toBe("number");
+          const input = Object.keys(
+            endpoint["GET /api/reviews/:review_id"].exampleResponse
+          );
+          const expectedOutput = Object.keys(response.body);
+          expect(input).toMatchObject(expectedOutput);
+        });
+      });
+  });
+  test("/api/reviews", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then((response) => {
+        response.body.reviews.forEach((review) => {
+          expect(typeof review.review_id).toBe("number");
+          expect(typeof review.title).toBe("string");
+          expect(typeof review.created_at).toBe("string");
+          expect(typeof review.votes).toBe("number");
+          expect(typeof review.comment_count).toBe("string");
+          expect(typeof review.owner).toBe("string");
+          expect(typeof review.category).toBe("string");
+          expect(typeof review.review_img_url).toBe("string");
+          expect(typeof review.designer).toBe("string");
+          const input = Object.keys(
+            endpoint["GET /api/reviews"].exampleResponse
+          );
```

Comment:
No need to test against the endpoints.json file, your earlier tests on the /api endpoint should hopefully have sorted that for you